### DISCUSSION TITLE
Changes required (minor) | Необходимые изменения (легкие)

### DISCUSSION THREAD

**Discussion by baccaraaa** [stoatchat/discussions#801] [Feature Suggestions]
- Зеленый кружок вокруг говорящего
- Вход по одному нажатию
- Отображение участников голосового канала
- Push-to-Talk
- Шумоподавление
- Возможность mute отдельных людей
- Возможность изменить громкость у отдельных участников

eng version
- Green circle around the speaker
- One-touch login
- Display the members of the voice channel
- Push-to-Talk
- Noise reduction
- Ability to mute individual people
- Ability to change the volume of individual participants

**Comment by minemokinesan** [comment]
Полностью поддерживаю предложение

**Reply by Kimber-user** [reply]
Коротко и по делу. Это необходимый минимум

**Reply by baccaraaa** [reply]
Благодарю за поддержку
